python/CMakeLists.txt

```diff
@@ -1 +1,15 @@
+# Python cmake, if testing enabled set pip to install test dependencies in venv
+if (KWIVER_ENABLE_TESTS)
+    if (PIP_COMMAND)
+        # list(SUBLIST PIP_COMMAND 0 1 py_pip_command)
```

Comment:
Are these going to be removed later?

Reply:
Yup, thought I had committed that before I pushed.